next/components/nav/Navbar.tsx

```diff
@@ -104,11 +107,15 @@ const Navbar: React.FC = () => {
                 <div className="hidden md:visible md:inline-flex">
                     <ul className="inline-flex flex-row flex-nowrap justify-between text-center text-lg">
                         {navItems.map((navItem, index) => (
-                            <NavItem key={index} {...navItem} onClickFunc={blurOnClick} />
+                            navItem.title === "Login" ? (
```

Comment:
Can you explain why we need just the Auth Button to be rendered differently than the other buttons? Just curious what is different about it

Reply:
Is it its onClick functionality?

Reply:
> Is it its onClick functionality?

The AuthButton is a separate component that I did not write, with additional logic to determine which text should be displayed and what the functionality should be, depending on auth state.

Reply:
I probably have the git blame on the auth button implementation.

For the record it was a quick and hacky component to get auth functioning.

Do you have any ideas after working with the code on how this section could be streamlined to remove special cases when rendering nav items?

Reply:
> I probably have the git blame on the auth button implementation.
> 
> For the record it was a quick and hacky component to get auth functioning.
> 
> Do you have any ideas after working with the code on how this section could be streamlined to remove special cases when rendering nav items?

No, not at this time. I can probably take a look later and see what I come up with.

Reply:
Ok, we can add that as a task, and in the meantime we'll get this PR merged and take care of the auth button implementation in a different PR